modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -14,11 +14,24 @@
 namespace cv {
 namespace gapi {
 
+/// Enum class specifies output format of stereo() function
 enum class StereoOutputFormat {
-    DEPTH_FLOAT16,
-    DEPTH_FLOAT32,
-    DISPARITY_FIXED16_11_5,
-    DISPARITY_FIXED16_12_4
+    DEPTH_FLOAT16, ///< Floating poinf 16 bit value, CV_16FC1
+    DEPTH_FLOAT32, ///< Floating poinf 32 bit value, CV_32FC1
+    DISPARITY_FIXED16_11_5, ///< 16 bit signed. 1 bit for sign,
+                            ///< 10 bits for integer,
```

Comment:
next 10 bits for integer part
and 5 bits for fractional part

Reply:
I don't understand this. If i have `-5,4` value then `-` is 1 bit, `5` is 10 bit and `4` is 5 bit?

Reply:
Exactly! 
First bit is a sing, next 10 is a integer path, last 5 is a fractional path

Reply:
I suggest:
```
16 bit signed. First bit for sign,
next 10 bits for integer part,
and 5 bits for fractional part.
```

Reply:
Done